modules/installation/pages/proxy-setup.adoc

```diff
@@ -220,19 +221,40 @@ If you do not use the reactivation key, you will need to re-register all the cli
 
 
 
-.Procedure: Replacing a Proxy and Keeping the Clients Registered
+.Procedure: Replacing a Traditional Proxy and Keeping the Clients Registered
 . Before starting the migration, save the data from the old proxy, if needed.
   Consider copying important or custom data to a central place that can also be accessed by the new proxy.
-. Shut down the proxy.
+. Shut down the old proxy.
 . Install a new {productname} Proxy.
   For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
 . In the {productname} {webui}, select the newly installed {productname} Proxy, and delete it from the systems list.
-. In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
+. In the {webui}, create a reactivation key for the old proxy system.
+  On the [guimenu]``System Details`` tab of the old proxy click [guimenu]``Reactivation``.
   Click [guimenu]``Generate New Key``, and make a note of the new key.
 . Register the proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
-  In the bootstrap script set the reactivation key with the [systemitem]``ACTIVATION_KEYS`` parameter.
-  For more information about using reactivation keys, see xref:client-configuration:activation-keys.adoc[].
-. Re-run the configure-proxy.sh as described in xref:installation:uyuni-proxy-setup.adoc#uyuni-proxy-setup-confproxy[].
+  In the bootstrap script, set the reactivation key with the [systemitem]``REACTIVATION_KEY`` parameter.
+. Restore the proxy from backup.
+
+For Salt proxies, you need to do some additional steps before you bootstrap the new proxy.
+
+
+
+.Procedure: Replacing a Salt Proxy and Keeping the Clients Registered
+. Before starting the migration, save the data from the old proxy, if needed.
+  Consider copying important or custom data to a central place that can also be accessed by the new proxy.
+. Shut down the old proxy.
+. In the {webui}, create a reactivation key for the old proxy system.
+  On the [guimenu]``System Details`` tab of the old proxy click [guimenu]``Reactivation``.
+  Click [guimenu]``Generate New Key``, and make a note of the new key.
+. In the {webui}, navigate to menu:Salt[Keys], locate the Salt key associated with the old proxy, and click btn:[delete].
+. Install a new {productname} Proxy.
+  For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
+. Register the new proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
+  In the bootstrap script, set the reactivation key with the [systemitem]``REACTIVATION_KEY`` parameter.
+. Restore the proxy from backup.
```

Comment:
I do not understand this step here, too.

Reply:
```suggestion
. Restore the proxy from the backup you made earlier.
```

Reply:
It is a little bit better, but stil not clear, IMO.  It is custom data you want to restore, not the proxies.  References such as "earlier" are not that user-friendly; with XML, you'd link back to the exact location.

Reply:
Feel free to go back over and edit. I don't like directional language either, generally speaking, but I'm struggling to see where the confusion is creeping in here.